### PR TITLE
Rename queryservice to querycoordinator

### PR DESCRIPTION
**Pull request by xige-16**
Signed-off-by: xige-16 <xi.ge@zilliz.com>

### CODE REVIEW

File: build/ci/jenkins/Jenkinsfile [hunk 1]
```diff
@@ -35,7 +35,7 @@ pipeline {
                     IMAGE_REPO = "dockerhub-mirror-sh.zilliz.cc/milvusdb"
                     DOCKER_BUILDKIT = 1
                     ARTIFACTS = "${env.WORKSPACE}/artifacts"
-                    MILVUS_HELM_BRANCH = "datacoord"
+                    MILVUS_HELM_BRANCH = "querycoord"
```
Comment (czs007) [inline build/ci/jenkins/Jenkinsfile:38]: what's this ?

Comment (xige-16) [reply build/ci/jenkins/Jenkinsfile:38]: @jeffoverflow

**Review by czs007 — APPROVED** [review]

**Pull request merged**